package.json

```
@@ -38,43 +38,44 @@
-    "astro-icon": "^1.1.1",
+    "@types/alpinejs": "^3.13.11",
+    "alpinejs": "^3.14.8",
+    "astro": "5.1.1",
+    "astro-embed": "^0.9.0",
+    "astro-icon": "^1.1.4",
     "astro-mastodon": "^0.3.2",
     "fuse.js": "^7.0.0",
     "github-slugger": "^2.0.0",
     "mdast-util-to-string": "^4.0.0",
-    "preact": "^10.23.2",
+    "react-icons": "^5.3.0",
```

Biome (1.9.4)

[error] The key react-icons was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)